collections/Hamilton/MS_Hamilton_15.xml

```diff
@@ -49,37 +47,267 @@
+                     <msItem><locus>(fols. 58r-59r)</locus> <incipit><quote>Conuertimini ad me in toto corde vestro</quote> <note>[gap left]</note> cor nostrum intra nos positum </incipit> <explicit>gaudium tuum et tristicia eque conuertantur adipsum ihesum christum dominum nostrum</explicit>
+                     <!-- similar incipit in Soccus sermones in Frankfurt (Main) MS. Barth. 119 f. 121 --></msItem>
+                     <msItem><note>(fols. 59-160v): Conrad de Brundelsheim, nos. 38-57, 61, 58-60, 62-86, [fol. 142v] 91-100.</note></msItem>
+                     <msItem>
+                        <locus>(fols. 160va-162vb)</locus>
+                        <incipit><quote>Ductus est Iesus in desertum</quote> ut supra. Dubitaret aliquis quomodo siue a quo spiritu ductus fuerit Iesus</incipit>
+                        <explicit>sitis extinccionem. Quod nobis prestare dignetur Iesus Christus Marie Filius. Sequitur.</explicit>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 163ra-164rb)</locus>
+                        <incipit><quote>Ecce nunc tempus acceptabile</quote> ubi supra. Nota quod est tempus de quo apocalipsis, Dedi illis tempus ut penitentiam agerent</incipit>
+                        <explicit>benedicat nos benediccione gratie in presenti et benediccione glorie in futuro</explicit>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 164rb-164vb)</locus>
+                        <incipit><quote>Ecce nunc tempus acceptabile.</quote> <sic><unclear>Oriuntur</unclear></sic>
+                           quinque de causis illud tempus est tempus acceptabile</incipit>
+                        <explicit>quia inuenitur quod perierat etc.</explicit>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 164vb-165ra)</locus>
+                        <incipit><quote>Ecce nunc tempus acceptabile</quote> Nunc licet cuncta fecerit bona tempore suo tamen aliquid est bonum in uno tempore quid non valeat in alio</incipit>
+                        <explicit>naute id est predicatores ad confessionem modo clama<supplied>n</supplied>t quod intrare debemus</explicit>
+                        <note>Cf. Schneyer VIII, p. 574, no. 33.</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 165ra-165va)</locus>
+                        <incipit><quote>Ecce nunc tempus acceptabile</quote> Nota super illo verbo Gal. ijo, In faciem restiti ei</incipit>
+                        <explicit>diuersitas intelligencie, quam nobis dominus</explicit>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 165va-169ra)</locus>
+                        <author key="person_89204668">Bertrandus de Turre</author> <title key="work_6170">Sermones de tempore et quadragesimales</title> <note>(nos. 105-106; Schneyer I, pp. 513-14)</note>
```

Comment:
Is it a problem to have works from other authors nested under an `<msItem>` that gives Conrad as the author of the whole?

Reply:
Possibly for some kinds of processing? Not for anything that we do at the moment on the website. It does I suppose indicate that these items circulated as Conrad.